ddtrace/contrib/elasticsearch/metadata.py

```diff
@@ -3,3 +3,4 @@
 TOOK = 'elasticsearch.took'
 PARAMS = 'elasticsearch.params'
 BODY = 'elasticsearch.body'
+STATUS = 'elasticsearch.status'
```

Comment:
rather than an extra metadata field, let's just use this key for status code
https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/ext/http.py#L16 

this keeps us consistent with our other integrations that use http